styles/components/_buttons.scss

```diff
@@ -0,0 +1,72 @@
+.button {
+  // Positioning
+  position: relative;
+
+  // Display & Box Model
+  border-radius: $border-radius;
+  border: 0;
+  padding: 1rem;
+  box-shadow: 0px 4px 4px rgba(0, 0, 0, 0.25);
+  min-width: 400px;
+
+  // Color
+  background-color: $color-primary;
+  color: $text-color-primary;
+
+  // Text
+  font-weight: 500;
+  font-size: $font-size-primary;
+  line-height: 1;
+
+  // Other
+  text-align: center;
+
+  &__answer {
+    margin: 1rem;
+    font-weight: 400;
+    background-color: $color-primary;
+    flex-grow: 1;
+    flex-basis: 40%;
+    &--success {
+      box-shadow: 0px 4px 4px rgba(0, 0, 0, 0.25),
+        0px 4px 30px #51fc00;
+      background-color: $color-correct;
+      font-weight: 600;
+    }
+    &--fail {
+      box-shadow: 0px 4px 4px rgba(0, 0, 0, 0.25),
+        0px 4px 30px $color-fail;
+      background-color: $color-fail;
+      font-weight: 600;
+    }
+    &--selected {
+      font-weight: 600;
+    }
+  }
+  &__icon {
+    position: absolute;
```

Comment:
How about using flex instead?

Reply:
Probably a good idea, especially when analyzing the coming responsivness